### ISSUE TITLE
List namespaces access required on cluster.

### ISSUE THREAD

**Issue by bartoncasey** [njust/kubelog#4]
Running the app fails with the error:
```
[2m2023-05-16T16:45:56.201628Z[0m [31mERROR[0m ThreadId(01) [2mkubelog::namespace_list_view[0m[2m:[0m Failed to load namespaces: ApiError: namespaces is forbidden: User "system:serviceaccount:<my namespace>:<my service account>" cannot list resource "namespaces" in API group "" at the cluster scope: Forbidden (ErrorResponse { status: "Failure", message: "namespaces is forbidden: User \"system:serviceaccount:<my namespace>:<my service account>\" cannot list resource \"namespaces\" in API group \"\" at the cluster scope", reason: "Forbidden", code: 403 })
```

It shouldn't be required to be able to list the namespaces. The namespace from the context, or an explicitly entered namespace, could be used.

**Comment by njust** [comment]
Hi, thanks for reporting this. You are right, cluster and namespace selection should be optional.

**Comment by njust** [comment]
Fixed in Version 0.4.0
